nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -360,4 +387,39 @@ private Type.ClassType typeWithPreservedAnnotations(ParameterizedTypeTree tree)
             type.getEnclosingType(), com.sun.tools.javac.util.List.from(newTypeArgs), type.tsym);
     return finalType;
   }
+
+  public void checkTypeParameterNullnessForAssignabilityForConditionalExpression(
```

Comment:
We should write some javadoc describing what check this method is doing.  Also I think the following is a better name:
```suggestion
  public void checkTypeParameterNullnessForConditionalExpression(
```